backend/users/Users.ts

```diff
@@ -0,0 +1,32 @@
+import { sequelize } from "../DB.js";
+
+export async function findOrCreateLocalUser(email) {
+  const users = await sequelize.models.User.findAll({
+    where: {
+      email: email,
+    },
+  });
+
+  let localUser = users.length > 0 ? users[0] : null;
+
+  if (!localUser) {
+    localUser = await sequelize.models.User.create({
+      firstName: "localDevFirstName",
+      lastName: "localDevLastName",
+      email: email,
+      password: "localDevPassword",
```

Comment:
This should be fine since it only works in development (usually putting passwords in code is not good)